src/components/Pagination.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+type PaginationProps = {
+  currentPage: number
+  data: ResponseDataType
+  setPage: (page: number) => void
```

Comment:
```suggestion
  setCurrentPage: (page: number) => void
```

matching up the names of state will reduce complexity